va/va.go

```diff
@@ -45,6 +45,13 @@ const (
 	//   PEBBLE_VA_NOSLEEP=1 pebble
 	noSleepEnvVar = "PEBBLE_VA_NOSLEEP"
 
+	// sleepTimeEnvVar defines the environment variable name used to set the time
+	// VA should sleep between validation attempts (if not disabled). Set this
+	// e.g. to 5 when you invoke Pebble if you wish the delays to be between 1
```

Comment:
I think this comment should say "between 0 and 5 seconds" and "instead of between 0 and 15 seconds" since as you point out in your PR description a sleep of 0s may occur.

Reply:
I've also adjusted the README.md, which also said "(from 1 to 15)".